src/test/java/ru/praktikum/scooter/ValidateCourierLoginTest.java

```diff
@@ -0,0 +1,40 @@
+package ru.praktikum.scooter;
+
+import io.restassured.response.ValidatableResponse;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class ValidateCourierLoginTest {
+
+    private final Courier courier;
+    private final int expectedStatusCode;
+    private final String expectedErrorMessage;
+
+    public ValidateCourierLoginTest(Courier courier, int expectedStatusCode, String expectedErrorMessage) {
+        this.courier = courier;
+        this.expectedStatusCode = expectedStatusCode;
+        this.expectedErrorMessage = expectedErrorMessage;
+    }
+
+    @Parameterized.Parameters
+    public static Object[][] getTestData() {
+        return new Object[][]{
+                {Courier.getLoginOnly(), 400, "Недостаточно данных для входа"},
```

Comment:
Твой тест падает с очень не хорошей ошибкой:
_Cannot invoke the path method because no content-type was present in the response and no default parser has been set.
You can specify a default parser using e.g.:
RestAssured.defaultParser = Parser.JSON;_
Со стороны это выглядит как косяк тестов, а не косяк сервиса. А это не так. Такая ситуация происходит из-за того, что наш сервер виснет не отдает тело ответа, но зато он отдает статус код. Если проверка на статус код делалась бы раньше проверки body, то тест бы падал с красивой ошибкой статус кода и вопросов бы не было. 
Поэтому я всегда рекомендую первым делом проверять статус код, а потом уже все остальное.